styletesla.css

```diff
@@ -1,97 +1,774 @@
+  margin-right: 30px;
 }
+
 .main {
-	background: url('img/tesla-wordmark.png') top no-repeat;
-	display: flex;
-	align-items: center;
-	flex-direction: column;
+  background: url('img/tesla-wordmark.png') top no-repeat;
+  background-size: contain;
+  display: flex;
+  align-items: center;
+  flex-direction: column;
 }
 
-.tesla{ 
-	/*width: 67%;*/
-	max-width: 968px;
+.main__title {
+  font: 9vh/11vh montserrat, sans-serif;
+  font-weight: 600;
+  margin: 0;
+}
+
+.main__subtitle {
+  font: 15px/18px 'montserratmedium', sans-serif;
+  margin: 0;
+}
 
+.tesla{ 
+  max-width: 105vh;
 }
+
 .description {
-	/*padding: 0;
-	height: 120px;*/
-	width: 600px;
-	display: flex;
-	flex-direction: row;
-	justify-content: space-between;
+  display: flex;
+  flex-direction: row;
+}
+
+.speed,
+.road {
+  display: flex;
+  margin-right: 80px;
+}
+
+.wheel {
+  display: flex;
+  margin-right: 50px;
 }
 
+.number {
+  font: 35px/22px montserratsemibold, sans-serif;
+}
+
+.dimension {
+  font: 20px/22px montserratsemibold, sans-serif;
+}
 
-.speed, .road, .wheel {
-	display: flex;
- /* переписать классы*/
+.text_bottom {
+  font: 12px/15px montserratregular, sans-serif;
 }
+
+.wheel__desc {
+  font: 13px/16px montserratbold, sans-serif;
+  align-self: center;
+}
+
 .icon_speed {
-	align-self: center;
-	width: 46px;
-	height: 34px;
+  align-self: center;
+  width: 53px;
+  height: 41px;
+  margin-right: 14px;
 }
+
 .icon_road {
-	width: 29px;
-	height: 45px;
-	align-self: center;
+  width: 29px;
+  height: 57px;
+  align-self: center;
+  margin-right: 14px;
+}
+
+.icon_redwheel {
+  display: none;
 }
 
 .icon_wheel {
-	width: 22px;
-	height: 22px;
-	align-self: center;
+  width: 22px;
+  height: 22px;
+  margin-right: 7px;
+  align-self: center;
+}
+
+.btn__order {
+  width: 160px;
+  height: 40px;
+  border: 2px solid #e51937;
+  align-self: center;
+  color: #e51937;
+  background: white;
+  font: 13px/16px montserratbold, sans-serif;
+  cursor: pointer;
+}
+
+.scrool {
+  cursor: pointer;
+}
+
+@media (max-width: 1000px) {
+  .logo {
+    padding-left: 0;
+  }
+}
+
+@media (max-width: 1100px) {
+  .mainpage {
+    height: 920px;
+  }
+
+  .menu,
+  .navigation {
+    display: none;
+  }
+
+  .menumobile {
+    height: 50px;
+    margin-right: 30px;
+    display: -webkit-box;
+    display: flex;
+    -webkit-box-orient: vertical;
+    -webkit-box-direction: normal;
+    flex-direction: column;
+    -webkit-box-pack: center;
+    justify-content: center;
+    cursor: pointer;
+  }
+
+  .menumobile span {
+    width: 35px;
+    margin: 3px auto;
+    display: block;
+    height: 3px;
+    -webkit-transform: rotate(0deg);
+    transform: rotate(0deg);
+    background: #e51937;
+  }
+
+  .header {
+    height: 83px;
+    margin-bottom: 0;
+  }
+
+  .main {
+    background: url('img/tesla-wordmark.png') no-repeat;
+    background-position-y: 6vh;
+    background-size: contain;
+  }
+
+  .main__subtitle {
+    margin-bottom: 12px;
+  }
+
+  .tesla { 
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected newline after "{" of a multi-line block (block-opening-brace-newline-after)](https://app.codacy.com/gh/WebHeroSchool/Tesla-12.04/pullRequest?prid=5512124)